packages/flex-plugin-scripts/src/scripts/pre-script-check.ts

```diff
@@ -175,6 +175,23 @@ export const _setPluginDir = (...args: string[]): void => {
   }
 };
 
+/**
+ * Compares the CLI and the plugin for flex-plugin-scripts
+ * @private
+ */
+export const _comparePluginAndCLIVersions = (): void => {
+  const cliPackageJson = readPackageJson(join(getCliPaths().dir, packageJsonName));
+  const pluginPackageJson = readPackageJson(join(getCwd(), packageJsonName));
+  const cliVersion = packageDependencyVersion(cliPackageJson, '@twilio-labs/plugin-flex');
+  const pluginVersion = packageDependencyVersion(pluginPackageJson, 'flex-plugin-scripts');
+
+  if (cliVersion && pluginVersion && semver.clean(cliVersion) !== semver.clean(pluginVersion)) {
+    logger.warning(
+      `You are using CLI Version \'${cliVersion}\' with flex-plugin-scripts version \'${pluginVersion}\' for plugin \'${pluginPackageJson.name}\'.`,
```

Comment:
We should add an action item, something like `Consider upgrading your CLI` or `Consider upgrading your flex-plugin-scripts` whatever that makes them the same. Confirm with Ruma on the exact wording